docs/guides/4-custom-rulesets.md

```diff
@@ -402,38 +402,51 @@ Rulesets can then reference aliases in the [given](#given) keyword, either in fu
 
 Previously Spectral supported exceptions, which were limited in their ability to target particular rules on specific files or parts of files, or changing parts of a rule. Overrides is the much more powerful version of exceptions, with the ability to customize ruleset usage for different files and projects without having to duplicate any rules.
 
-Overrides can be used to:
+Overrides can be used to apply rulesets on:
 
-- Override rulesets to apply on particular files/folders `files: ['schemas/**/*.draft7.json']`
-- Override rulesets to apply on particular JSONPath's `files: ['**#/components/schemas/Item']`
-- Override rulesets to apply on particular formats `formats: [jsonSchemaDraft7]`
+- Particular formats `formats: [jsonSchemaDraft7]`
+- Particular files/folders `files: ['schemas/**/*.draft7.json']`
+- Particular elements of files `files: ['**#/components/schemas/Item']`
 - Override particular rules
 
 **Example**
 
+```yaml
+overrides:
+  formats:
+    - json-schema-draft7
+  files:
+    - schemas/**/*.draft7.json
+  rules:
+    valid-number-validation:
+      given:
+        - $..exclusiveMinimum
+        - $..exclusiveMaximum
+      then:
+        function: schema
+        functionOptions:
+          type: number
+```
+
+To apply an override to particular elements of files, combine a glob for a filepath
+with a [JSON Pointer](https://github.com/janl/node-jsonpointer) after the anchor, i.e.:
+
 ```yaml
 overrides:
   - files:
-      - schemas/**/*.draft7.json
-    formats:
-      - json-schema-draft7
+      - "legacy/**/*.oas.json#/paths"
     rules:
-      valid-number-validation:
-        given:
-          - $..exclusiveMinimum
-          - $..exclusiveMaximum
-        then:
-          function: schema
-          functionOptions:
-            type: number
+      some-inherited-rule: "off"
 ```
 
-One can also combine a glob for a filepath with a JSONPath after the anchor, i.e.:
+JSON Pointers have a different syntax than JSON Paths used in the `given` component of a rule.
+In JSON Pointers, path components are prefixed with a "/" and then concatenated to form the pointer.
+Since "/" has a special meaning in JSON pointer, it must be encoded as "~1" when it appears in a component, and "~" must be encoded as "~0".
```

Comment:
It would have been nice to point to an online JSON pointer evaluator, as we have for JSON Paths, but I could not find a good one.

Reply:
https://www.jsonquerytool.com/ has `json pointer` to choose from
<img width="498" alt="image" src="https://user-images.githubusercontent.com/9273484/154965798-7df34fd5-3f4d-415c-8991-e20a4d34c391.png">